### PR TITLE
Update fv3 component in ufs-s2s-model

### LINKED ISSUE #43: Porting ufs-s2s-model to dell
ufs-s2s-model needs to be ported to wcoss dell as phase1/2 are going away

### LINKED ISSUE #120: Update FV3 component to #def9be1 (6/5/2020) version

### PR DESCRIPTION
**Pull request by binli2337**
1. Update the FV3 component to #def9be1 (6/5/2020) version.
2. Generate new baselines on Hera and Orion.
3. Generate new baselines on wcoss_dell_p3.

close #43 
close #120

### CODE REVIEW

File: parm/input.benchmark_ccpp.nml.IN [hunk 1]
```diff
@@ -196,6 +196,7 @@
   iopt_stc     = 1
   debug        = .false.
   nstf_name    = 0,0,0,0,0
+  frac_grid    = .false.
```
Comment (DeniseWorthen) [inline parm/input.benchmark_ccpp.nml.IN]: I think it better if this were defined as a variable in default_vars. This is what I had added for the upcoming CMEPS commit which has fractional grid capabilities:

#default settings for fractional grid
export FRAC_GRID='.F.'
export CPLMODE='nems_orig'

Then we'd have frac_grid = @[FRAC_GRID] in the namelists.

Comment (junwang-noaa) [reply parm/input.benchmark_ccpp.nml.IN]: I think it is good to set frac_grid = @[FRAC_GRID], and then in default_vars.sh, define FRAC_GRID=.false. for this commit.

Comment (binli2337) [reply parm/input.benchmark_ccpp.nml.IN]: Updates have been made to the following 3 files:
parm/input.benchmark_ccpp.nml.IN
parm/input.mom6_ccpp.nml.IN
tests/default_vars.sh

**Comment by binli2337** [comment]
PR #122 is ready for review.

**Review by DeniseWorthen — APPROVED** [review]

**Review by junwang-noaa — APPROVED** [review]
Looks good.

**Pull request merged**